src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -331,7 +385,60 @@
   end
 
   describe '#setup_codemirror_editor' do
-    skip('Please add some tests')
+    it 'sets the style for the editor' do
```

Comment:
I think mocking is an essential part when it comes to testing. I personally would keep it as before but I'm up to other opinions.